### PR TITLE
Fix Event Imports

### LINKED ISSUE #767: Implement event-import API to events/import
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

### PR DESCRIPTION
**Pull request by YogeshSharma0201**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:

Event Import is working now and previous import jobs are being displayed.
Only zip files can be used for importing as of now and other importing methods are not implemented on backend. 

#### Changes proposed in this pull request:

- Add model for import jobs
- Added neccessary actions and function to handle import 

import

Fixes #767
Backend Changes PR: fossasia/open-event-server#5514

### CODE REVIEW

File: app/components/events/event-import-section.js [hunk 1]
```diff
@@ -26,9 +28,62 @@ export default Component.extend(FormMixin, {
...
+          var ext = file.name.split('.');
+          ext = ext[ext.length - 1].toLowerCase();
+          if (ext === 'xml') {
+            endpoint = 'import/pentabarf';
+          } else if (ext === 'ical') {
+            endpoint = 'import/ical';
+          }
+          data.append('file', file);
+
+          $('#import_status').text('Uploading file.. Please don\'t close this window');
```
Comment (CosmicCoder96) [inline app/components/events/event-import-section.js]: Hey, check out the code for how the export of events his handled, we don't use JQuery selectors directly in the codebase, thank you!

**Comment by YogeshSharma0201** [comment]
@CosmicCoder96 i have removed jquery usage.

File: app/routes/events/import.js [hunk 2]
```diff
@@ -3,5 +3,14 @@ import Route from '@ember/routing/route';
 export default Route.extend({
   titleToken() {
     return this.get('l10n').t('Import');
+  },
+  async model() {
+
+    const data = await this.get('store').findAll('importJob');
```
Comment (CosmicCoder96) [inline app/routes/events/import.js]: This is not a valid query. There is no importJob model, please try importing the event yourself, and request a review when it works.

Comment (YogeshSharma0201) [reply app/routes/events/import.js]: I have implemented the importJob model both in backend and fronted. Please check fossasia/open-event-server#5514.

Comment (YogeshSharma0201) [reply app/routes/events/import.js]: I have tested it locally, it works.

**Comment by YogeshSharma0201** [comment]
@CosmicCoder96 Issue A would also require changes on server side, see changes in file app/models/event.py and app/api/helpers/utilities.py in server side pr, these changes are required for successfully importing events. Should i still open a separate pr?

**Comment by CosmicCoder96** [comment]
@YogeshSharma0201 Yes, please still open a separate PR. Also, we can't have the mentioned changes in the utilities.py, it will break several other things, try to find a workaround to that.

**Comment by kushthedude** [comment]
@YogeshSharma0201 Still on it ?

**Comment by YogeshSharma0201** [comment]
@kushthedude this PR is complete and updated (no merge conflicts)

**Comment by kushthedude** [comment]
@YogeshSharma0201 thats great , @CosmicCoder96 review please.

**Pull request merged**